docs/features/private-wlan.rst

```diff
@@ -4,6 +4,14 @@ Private WLAN
 It is possible to set up a private WLAN that bridges the WAN port and is separated from the mesh network.
 Please note that you should not enable ``mesh_on_wan`` simultaneously.
 
+The private WLAN is encrypted via WPA2 by default. On devices with enough flash and a supported radio,
```

Comment:
```suggestion
The private WLAN feature uses WPA2 encryption by default. On devices with enough flash and a supported radio,
```

Reply:
Altered this part according to @NeoRaider s previous feedback (which was overwritten by accident).